README.md

```diff
@@ -1,44 +1,26 @@
-# Standup
+# Retro
 
-## TODO
-
-1. sockets for column create, delete, rename
-* [x] Ability to rename columns
-* [x] socket wire ups
-
-2. create a board (index -> create new button -> new board with unique URL)
-* [x] board is held in memory - don't overthink it
-* [x] board stores name, context
-* [x] UUID for column ids - board stores column IDs
-* [x] socket on "connection" passes down name/context/boards if you create/join
-* [x] "create new board" from step 2 creates & passes initial columns to board on creation
+A tool for running retros.
 
-3. store data so people joining URL see what's already been created
-* [x] store card data in memory on server
-* [x] socket on "connection" passes columns and cards down to client
+<img width="945" alt="Screenshot" src="https://user-images.githubusercontent.com/3317231/71594346-72821e80-2aec-11ea-836e-0da5246fe544.png">
 
-4. editable board traits
-* [x] title saves to server
-* [ ] description saves to server
+## Getting Started
 
-5. make it public!
-* [x] figure out ngrok or some other system
+1. `npm install` // Installs package dependencies.
+1. Create an environment variable called `RETRO_SECRET` and store its value somewhere safe. It must be able to be accessed via `process.env.RETRO_SECRET` in the server code. If you do not do this, you will see an error when you try to access the app after running the steps below.
```

Comment:
is there any way to automate this in one of the `start` commands? Or a separate npm script to create and store a secret if one doesn't exist?